### ISSUE TITLE
Blurry font on macOS

### ISSUE THREAD

**Issue by yoachim** [JabRef/jabref#8387]
### JabRef version

5.4 (latest release)

### Operating system

macOS

### Checked with the latest development build

- [X] I made a backup of my libraries before testing the latest development version.
- [X] I have tested the latest development version and the problem persists

### Steps to reproduce the behaviour

Just loading up the latest version on my mac, the font look terrible. I assume this is some java sillyness. 

I tried changing the Info.plist file line:
```
<key>NSHighResolutionCapable</key>
<string>true</string>
```
to
```
<key>NSHighResolutionCapable</key>
<true/>
```
but that didn't help.

Looks like this might be the same issue as #6757

### Appendix

![Screen Shot 2022-01-04 at 11 21 48 AM](https://user-images.githubusercontent.com/3673733/148112462-4fbf706c-1621-4556-ab2f-ef061a1a6d51.png)

**Comment by Siedlerchr** [comment]
As  I mentioned in the linked issue, looks still good on my MacBook (Big Sur).  Might be this java bug? https://bugs.openjdk.java.net/browse/JDK-8236689
Maybe you could also write down some more details of your mac

**Comment by yoachim** [comment]
Yes, I think it's that java bug. Looks like the issue is that it renders fine on my laptop screen but fails if I put it on an external monitor.

**Comment by Siedlerchr** [comment]
It seems it's fixed with openjfx18. We will upgrade to that once it's officially released  (probably it will be released around March). It will also (hopefully) contain some other mac fixes.

**Comment by yoachim** [comment]
> @yoachim Can you please test the latest dev version? We just upgraded to jfx 18 https://builds.jabref.org/main/

Font looks great on the external monitor. It can't open or import my .bib file, so that's an issue.

**Comment by Siedlerchr** [comment]
@yoachim  Ugh .Can you please check if there are any errors present in the log? Can you also try to open your bib file with TextEditor and check the encoding?

**Comment by Siedlerchr** [comment]
When you try to import your bib file into a new library, JabRef should log an error to the log and tell you which entry is broken or where it is broken
Otherwise try to copy your bib file (aka make a backup) and resave it as utf8
